alicloud/service_alicloud_drds.go

```diff
@@ -0,0 +1,350 @@
+	raw, err := s.client.WithDrdsClient(func(drdsClient *drds.Client) (interface{}, error) {
+		return drdsClient.CreateReadOnlyAccount(req)
+	})
+	resp, _ := raw.(*drds.CreateReadOnlyAccountResponse)
+
+	return resp, err
+}
+
+func (s *DrdsService) DescribeReadOnlyAccount(drdsInstanceId string, dbName string) (response *drds.DescribeReadOnlyAccountResponse, err error) {
+	req := drds.CreateDescribeReadOnlyAccountRequest()
+	req.DrdsInstanceId = drdsInstanceId
+	req.DbName = dbName
+	//resp, err := client.drdsconn.DescribeReadOnlyAccount(req)
+	raw, err := s.client.WithDrdsClient(func(drdsClient *drds.Client) (interface{}, error) {
+		return drdsClient.DescribeReadOnlyAccount(req)
+	})
+	resp, _ := raw.(*drds.DescribeReadOnlyAccountResponse)
+
+	return resp, err
+}
+
+func (s *DrdsService) ModifyReadOnlyAccountPassword(drdsInstanceId string, dbName string, accountName string, originPassword string, newPassword string) (response *drds.ModifyReadOnlyAccountPasswordResponse, err error) {
+	req := drds.CreateModifyReadOnlyAccountPasswordRequest()
+	req.DbName = dbName
+	req.DrdsInstanceId = drdsInstanceId
+	req.AccountName = accountName
+	req.OriginPassword = originPassword
+	req.NewPasswd = newPassword
+	//resp, err := client.drdsconn.ModifyReadOnlyAccountPassword(req)
+	raw, err := s.client.WithDrdsClient(func(drdsClient *drds.Client) (interface{}, error) {
+		return drdsClient.ModifyReadOnlyAccountPassword(req)
+	})
+	resp, _ := raw.(*drds.ModifyReadOnlyAccountPasswordResponse)
+
+	return resp, err
+}
+
+func (s *DrdsService) RemoveReadOnlyAccount(drdsInstanceId string, dbName string, accountName string) (response *drds.RemoveReadOnlyAccountResponse, err error) {
+	req := drds.CreateRemoveReadOnlyAccountRequest()
+	req.DrdsInstanceId = drdsInstanceId
+	req.DbName = dbName
+	req.AccountName = accountName
+	//resp, err := client.drdsconn.RemoveReadOnlyAccount(req)
+	raw, err := s.client.WithDrdsClient(func(drdsClient *drds.Client) (interface{}, error) {
+		return drdsClient.RemoveReadOnlyAccount(req)
+	})
+	resp, _ := raw.(*drds.RemoveReadOnlyAccountResponse)
+
+	return resp, err
+}
+
+func (s *DrdsService) CreateDrdsAccount(drdsInstanceId string, dbName string, password string) (response *drds.CreateDrdsAccountResponse, err error) {
+	req := drds.CreateCreateDrdsAccountRequest()
+	req.DrdsInstanceId = drdsInstanceId
+	req.DbName = dbName
+	req.Password = password
+	//resp, err := client.drdsconn.CreateDrdsAccount(req)
+	raw, err := s.client.WithDrdsClient(func(drdsClient *drds.Client) (interface{}, error) {
+		return drdsClient.CreateDrdsAccount(req)
+	})
+	resp, _ := raw.(*drds.CreateDrdsAccountResponse)
+
+	return resp, err
+}
+
+func convertTypeValue(returnedType string, rawType string) InstanceType {
+	var i InstanceType
+	returnedInstanceType := InstanceType(returnedType)
+	switch returnedInstanceType {
+	case PrivateType_:
+		i = PrivateType
+	case PublicType_:
+		i = PublicType
+	}
+	return i
```

Comment:
Please remove the useless methods.

Reply:
publicType will not sale

Reply:
I mean please remove the useless methods in this file

Reply:
Staying first